baselines/marl_benchmark/README.md

```diff
@@ -3,19 +3,19 @@
 This directory contains the scenarios, training environment, and agents used in the CoRL20 paper: [SMARTS: Scalable Multi-Agent ReinforcementLearning Training School for Autonomous Driving](...).
 
 **Contents,**
-- `agents/`: YAML files and some RLlib-based policy implementations
-- `metrics/`: Class definition of metrics (default by a basic Metric class)
-- `networks/`: Custom network implementations
-  - `communicate.py`: Used for Networked agent learning
-- `scenarios/`: Contains three types of scenarios tested in the paper
-- `wrappers/`: Environment wrappers
-- `evaluate.py`: The evaluation program
-- `run.py`: Executes multi-agent training
+- `marl_benchmark/agents/`: YAML files and some RLlib-based policy implementations
+- `marl_benchmark/metrics/`: Class definition of metrics (default by a basic Metric class)
+- `marl_benchmark/networks/`: Custom network implementations
+  - `marl_benchmark/communicate.py`: Used for Networked agent learning
+- `marl_benchmark/scenarios/`: Contains three types of scenarios tested in the paper
+- `marl_benchmark/wrappers/`: Environment wrappers
+- `marl_benchmark/evaluate.py`: The evaluation program
+- `marl_benchmark/run.py`: Executes multi-agent training
```

Comment:
I feel like prepending the new folder name just adds noise to the documentation.